src/sentry/models/integrationfeature.py

```diff
@@ -11,6 +11,8 @@ class Feature(object):
     ISSUE_LINK = 1
     STACKTRACE_LINK = 2
     EVENT_HOOKS = 3
+    PROJECT_MANAGEMENT = 4
+    INCIDENT_MANAGEMENT = 5
```

Comment:
@NisanthanNanthakumar So I asked @jonesphillip to come up with updated categories for our integrations. He wants to add one for "feature flag". See the card: https://getsentry.atlassian.net/browse/API-884